packages/site-kit/src/lib/styles/tokens.css

```diff
@@ -1,205 +1,9 @@
-	--sk-raised-active-color: var(--sk-back-6) var(--sk-back-5) var(--sk-back-5) var(--sk-back-6);
-	--sk-raised-active-width: 2px 1px 1px 2px;
-
 	/* this is necessary for elements with filters on safari, because safari is a uniquely awful piece of software.
 	   the 0.01 prevents Vite from optimising it to `translate(0, 0)`, which doesn't fix anything */
 	--safari-fix: translate3d(0, 0, 0.01);
-
-	/* set at 1.75x so if you zoom +/-10% from common settings it won't switch your font */
-	@media (min-resolution: 1.75x) {
-		--sk-font-family-body: 'EB Garamond', Georgia, serif;
-		--sk-font-size-body: 2.2rem;
-		--sk-font-size-body-small: 2rem;
-	}
-
-	&.font-boring {
-		--sk-font-family-body: 'Atkinson Hyperlegible', sans-serif;
-		--sk-font-size-body: 1.9rem;
-		--sk-font-size-body-small: 1.7rem;
-
-		p code {
-			font-size: 0.75em;
-		}
-	}
-
-	&.dark {
-		color-scheme: dark;
-		-webkit-font-smoothing: antialiased;
-
-		--sk-back-hue: 220;
-
-		--sk-back-1: hsl(var(--sk-back-hue), 15%, 8%);
-		--sk-back-2: hsl(var(--sk-back-hue), 15%, 15%);
-		--sk-back-3: hsl(var(--sk-back-hue), 15%, 12%);
-		--sk-back-4: hsl(var(--sk-back-hue), 15%, 22%);
-		--sk-back-5: hsl(var(--sk-back-hue), 15%, 25%);
-		--sk-back-6: hsl(var(--sk-back-hue), 15%, 32%);
-		--sk-back-translucent: hsla(0, 0%, 100%, 0.1);
-		--sk-theme-1-hsl: 12, 94%, 62%;
-		--sk-text-1: hsl(var(--sk-back-hue), 10%, 90%);
-		--sk-text-2: hsl(var(--sk-back-hue), 10%, 80%);
-		--sk-text-3: hsl(var(--sk-back-hue), 10%, 65%);
-		--sk-text-4: hsl(var(--sk-back-hue), 10%, 45%);
-		--sk-text-translucent: hsla(0, 0%, 100%, 0.9);
-		--sk-scrollbar: rgba(255, 255, 255, 0.3);
-		--sk-shadow: drop-shadow(1px 2px 16px rgba(0, 0, 0, 0.5));
-
-		--sk-theme-1-variant: hsl(15, 100%, 35%);
-
-		--sk-text-warning-hsl: 32, 67%, 56%;
-
-		--sk-code-base: hsl(45, 7%, 75%);
-		--sk-code-comment: hsl(0, 0%, 55%);
-		--sk-code-keyword: hsl(204, 88%, 65%);
-		--sk-code-function: hsl(19, 67%, 75%);
-		--sk-code-string: hsl(41, 37%, 68%);
-		--sk-code-number: hsl(120, 100%, 25%);
-		--sk-code-template-string: hsl(2, 80%, 47%);
-		--sk-code-tags: var(--sk-code-function);
-		--sk-code-important: var(--sk-code-string);
-
-		--sk-code-diff-base: hsla(0, 0%, 100%, 0.5);
-		--sk-code-diff-inserted: hsl(120, 46%, 48%);
-		--sk-code-diff-removed: hsl(2, 64%, 70%);
-
-		/* overrides */
-		--shiki-color-background: var(--sk-back-3);
-
-		/* raised buttons */
-		--sk-raised-color: var(--sk-back-5) var(--sk-back-3) var(--sk-back-3) var(--sk-back-5);
-		--sk-raised-hover-color: var(--sk-back-6) var(--sk-back-3) var(--sk-back-3) var(--sk-back-6);
-		--sk-raised-active-color: var(--sk-back-3) var(--sk-back-6) var(--sk-back-6) var(--sk-back-3);
-	}
-}
-
-@media screen and (min-width: 480px) {
-	:root {
-		--sk-page-padding-side: 3.2rem;
-	}
-}
-
-@media screen and (min-width: 800px) {
-	:root {
-		--sk-font-size-h1: 5.4rem;
-		--sk-page-padding-top: 8rem;
-		--sk-page-padding-side: 4.8rem;
-		--sk-secondary-nav-height: 6rem;
-	}
```

Comment:
was the split really necessary? There's already a TODO in `colours.css` about "move this somewhere else?", and it's only going to get tricker as time goes. Like, we may have a `--border-1` setting for `1px solid var(--...)` at some point, and then the question is where does that even go. That would be much easier if everything was in one file, and then you could just put it next to the border stuff.

Reply:
I find it a lot more manageable, yeah. I resolved that TODO by moving the raised button styles into `buttons.css` (even if they're not always used with a `<button>`, the things that use those styles are always 'buttons')